### DISCUSSION TITLE
Why are two endorsers built for Traction?

### DISCUSSION THREAD

**Discussion by amanji** [bcgov/traction#1014] [Q&A]
The docker compose file contains configs for 2 endorser agents and API's. What purpose do each of the endorsers serve?

See: 
* https://github.com/bcgov/traction/blob/b2f61e6d7dfe4ae39ff06f68d91fb279d19923e7/scripts/docker-compose.yml#L280
* https://github.com/bcgov/traction/blob/b2f61e6d7dfe4ae39ff06f68d91fb279d19923e7/scripts/docker-compose.yml#L310
* https://github.com/bcgov/traction/blob/b2f61e6d7dfe4ae39ff06f68d91fb279d19923e7/scripts/docker-compose.yml#L352

**Comment by loneil** [comment]
This is to give an ecosystem that can use/validate the mult-ledger functionality for multitenancy that was added to ACA-Py
